build/Dockerfile

```diff
@@ -63,6 +63,11 @@ WORKDIR /build/windows-machine-config-operator/cloud-provider-azure/
 COPY cloud-provider-azure/ .
 RUN GOOS=windows go build -o azure-cloud-node-manager.exe ./cmd/cloud-node-manager
 
+# Build ecr-credential-provider
```

Comment:
Changes may be required in the [mid-stream dockerfile](https://gitlab.cee.redhat.com/openshift-winc-midstream/openshift-winc-midstream/-/blob/rhaos-4.16-rhel-9/distgit/containers/windows-machine-config-operator/Dockerfile?ref_type=heads#L1) as well.